core/contracts/test/TestableTest.sol

```diff
@@ -0,0 +1,8 @@
+pragma solidity ^0.5.0;
+
+import "../Testable.sol";
+
+// TestableTest is derived from the abstract contract Testable for testing purposes.
+contract TestableTest is Testable {
+    constructor(bool _isTest) public Testable(_isTest) {}
```

Comment:
You might have to disable the linter on this line:
`// solhint-disable-next-line no-empty-blocks`

Reply:
ah, good point.